static/src/js/reducers/ui.js

```diff
@@ -195,26 +200,74 @@ const uiReducer = (state = initialState, action) => {
       }
     }
     case EXPORT_STARTED: {
-      return {
-        ...state,
-        export: {
-          isExportRunning: {
-            ...state.export.isExportRunning,
-            [action.payload]: true
+      const { type, format } = action.payload
+
+      if (type === 'collection') {
+        return {
+          ...state,
+          export: {
+            isExportRunning: {
+              ...state.export.isExportRunning,
+              collection: {
```

Comment:
Can we use the type as a variable on lines 211 and 226 to avoid the duplicate code?